qa/rpc-tests/bigblocks.py

```diff
@@ -198,8 +214,22 @@ def run_test(self):
         # Time starts a little before fork activation time:
         set_node_times(self.nodes, FORK_TIME - 100)
 
-        # No supermajority
-        self.TestMineBig(False)
+        # No supermajority, vote2mb=1
+        self.TestMineBig(False, True)
+
+        # Create a block after the expiration date. This will be rejected 
+        # by the other nodes for being more than 2 hours in the future,
```

Comment:
> This will be rejected  by the other nodes

Is that being asserted? Will that start failing once the expiry date has arrived?